### ISSUE TITLE
Permission slip NOT turned in (visual indicator)

### ISSUE THREAD

**Issue by matthewerlemiller** [matthewerlemiller/projectgenesys#69]
When a member is searched, next to "status", add a red indicator that permission slip is not turned in.

Something like this:
![doc_red](https://cloud.githubusercontent.com/assets/8120005/14938554/13c8f678-0edc-11e6-9ee8-d0b792d728d9.png)
![doc_green](https://cloud.githubusercontent.com/assets/8120005/14938555/16871174-0edc-11e6-99f1-5b795c323fc9.png)

When a student has checked in a third time with no permission slip, a warning message should give leader this warning:

"Permission slip required for member check-in."

I would like a weekly emailed sent to every director with a list of students that do not have permission slips turned in with 3 or more visits, but that may be more of a postgres or shell command. Not sure yet. Suggestions?

**Explanation:**
When a member is first added to Visit, a permission is not required, but the leader hands a member a physical permission slip. The member is expected to take that permission slip home to be filled out by a parent. The member returns with a permission slip and gives it to the leader on shift that day. The leader turns in the permission slip to the director. The director of that center uses the permission slip to update the members profile, and checks off that the permission slip has been turned in.

**Comment by matthewerlemiller** [comment]
A note about this, I would say this request and kickout functionality are the two highest priorities. This was specifically asked for due to liability issues and protection they are seeking. This gives a method to ensure kids are getting permission slips filled out and turned in.

**Comment by austenpayan** [comment]
Totally doable! 

Regarding the weekly email, that sounds like a Cron-job or something similar. I am not familiar with doing a scheduled task like that at the moment. 

Though, for the moment, I could send emails on a member-by-member basis that is triggered on a "third" checkin without a permission slip. What do you think?

**Comment by matthewerlemiller** [comment]
Yes that would work! I can look into Cron job with Alex. The database
connection string may be the tricky part but we'll figure it out.

For now a member by member email works. I'm guessing you could use a table
associating centers with directors with email addresses?

**Comment by austenpayan** [comment]
yeah I can make that. From the admin, I can make a separate area where you can define directors' info and associate them with a center.

**Comment by austenpayan** [comment]
<img width="710" alt="screen shot 2016-05-11 at 7 56 50 pm" src="https://cloud.githubusercontent.com/assets/3936068/15202832/9ebd778e-17b2-11e6-8201-83111e353eee.png">
Permission slip indicators are live, I am going to keep this issue unresolved though as I create the email notification system. For now we just have the visual cues to the leaders of 'this member has no permission slip'

**Comment by austenpayan** [comment]
I also opted to only show an indicator when the member DOESN'T have a permission slip, in order to keep things minimal and less cluttered once most of them do have permission slips. Any problems with that?

**Comment by matthewerlemiller** [comment]
that's good for me! looks great. Thanks!
